core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.ts

```diff
@@ -30,18 +35,31 @@ export class DotEditPageToolbarComponent implements OnInit, OnChanges, OnDestroy
     showWhatsChanged: boolean;
     apiLink: string;
     pageRenderedHtml: string;
+    dotFavoritePageIconName = 'star_outline';
+
+    // TODO: Remove next line when total functionality of Favorite page is done for release
+    showFavoritePageStar: boolean;
 
     private destroy$: Subject<boolean> = new Subject<boolean>();
 
     constructor(
         private dotLicenseService: DotLicenseService,
         private dialogService: DialogService,
-        private dotMessageService: DotMessageService
+        private dotMessageService: DotMessageService,
+        private dotESContentService: DotESContentService,
+
+        // TODO: Remove next line when total functionality of Favorite page is done for release
+        private loggerService: LoggerService
     ) {}
 
     ngOnInit() {
+        // TODO: Remove next line when total functionality of Favorite page is done for release
+        this.showFavoritePageStar = this.loggerService.shouldShowLogs();
```

Comment:
What logs have to do here?

Reply:
nothing but I'm reusing `shouldShowLogs()` function from logs to ONLY show the star icon in DEV mode, so once it gets released this commit in demo, it won't break anything

Note: I added a `TODO` on every line that needs to be removed once the feature is finished and ready for release